docs/kcMigration/KeycloakMigration.md

```diff
@@ -0,0 +1,52 @@
+---
+description: KeyCloak (RH-SSO) Migration from Pathfinder to BCGov service
+resourceType: Documentation
+title: BC Government KeyCloak (RH-SSO) Service Migration Plan
+tags:
+- sso service migration
+- keycloak host name change
+---
+
+## KeyCloak Service Migration
+
+Hi team, the pathfinder KeyCloak SSO service is undergoing a transformation to an organizational BCGov service. For more information on this Single Sign-On service, refer to [here](https://developer.gov.bc.ca/BC-Government-SSO-Service-Definition).
```

Comment:
You are extremely vague about what that transformation is, and how it will affect the users of the service.  Would be awesome if you could be more specific.. maybe:
```suggestion
Hi team, the KeyCloak SSO service is finally removing its *Pathfinder* sticker and getting a new name.  This change does not affect the service offering but does change how you will access it.  The name is changing from sso.pathfinder.gov.bc.ca to oidc.gov.bc.ca.  For more information on this Single Sign-On service, refer to [here](https://developer.gov.bc.ca/BC-Government-SSO-Service-Definition).
```